Vizzuality/trase
# Feature/data view widget unit

## Asana

https://app.asana.com/0/1198886845396491/1199896185325402/f

## Description

Add suffix as the unit to the charts. We only need it on the default charts as the others have the unit implemented. Is important not to use caps as 't' is a different unit than 'T'

![image](https://user-images.githubusercontent.com/9701591/109666472-80f41c00-7b6f-11eb-8ad2-1fd897a78d3c.png)


## Testing instructions

Go to the data view of any context. You should be able to see the units on the lower right of the default charts. Change some indicator and units just to test
